### DISCUSSION TITLE
How to enable both remote and local endpoint?

### DISCUSSION THREAD

**Discussion by genkv** [immich-app/immich#8742] [Q&A]
Like many members of this community, I host the immich server and storage at my own home. I run a reverse proxy on a VPS as the endpoint, only to redirect the traffic back to my home network, and the only reason I do this is because my ISP doesn't provide a public IPv4 address.

This approach keeps immich functioning when I'm not home. But when I am home, which is 60% of the time, this back-and-force traffic obviously seems unnecessary when I can just use a local domain as the endpoint for maximum bandwidth. So is there anyway I can configure my Android App to enable both a remote and a local endpoint?

**Comment by genkv** [comment]
After doing some more searching, I learned that there's no easy way at least as of now. One workaround is to setup a DNS A record in my home router. But that seems a little hectic.

**Comment by Aralox** [comment]
FYI for those searching, this has been implemented:
https://github.com/immich-app/immich/issues/308
https://github.com/immich-app/immich/releases/tag/v1.122.0
